chargeio/infrastructure/Requestor.cs

```diff
@@ -75,9 +73,9 @@ public static string Delete(string url, string secretKey = null)
             }
         }
 
-        private static async Task<string> Delete(string url)
+        private static async Task<string> _Delete(string url, string secretKey)

```

Comment:
Why does this method start with an underscore? It should just be "Delete(...)" with the additional secretKet param, just like "Get(...)".